packages/grafana-ui/src/components/Alert/_Alert.scss

```diff
@@ -8,7 +8,6 @@
 .alert {
```

Comment:
I think this should be safe to remove, apart from the page-alert-list OFC.

Reply:
Any idea where I could move `page-alert-list` styles to, so this whole file could be deleted then?

Reply:
@Clarity-89 doing quick look at the code. Seems like the class name is passed to the angular wrapper for notifications list: https://github.com/grafana/grafana/blob/d04dce6a37baa21ceb35e3caf7a3644cf33cd4c7/public/views/index-template.html#L214

This wrapper renders https://github.com/grafana/grafana/blob/99411bf37a66f8a8d6720ee06c5849855533d424/public/app/core/components/AppNotifications/AppNotificationList.tsx component, which does not use className at all. Maybe this class is not used in the app at all?

Reply:
Thanks for the pointer! Seems like this file is a copy of https://github.com/grafana/grafana/blob/master/public/sass/components/_alerts.scss, so it can be safely deleted.